### PR TITLE
[SMALLFIX] PermissionCheckTest.java

### PR DESCRIPTION
**Pull request by JiangXiNJU**
[SMALLFIX] PermissionCheckTest.java
2017-10-08-JiangXiNJU

### CODE REVIEW

File: core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java [hunk 1]
```diff
@@ -73,8 +77,9 @@
 
 /**
  * Unit test for {@link FileSystemMaster} when permission check is enabled by configure
- * alluxio.security.authorization.permission.enabled=true.
+ * alluxio.security.authorization.permission.enabled=true...
  */
+
```
Comment (apc999) [inline core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java]: can you remove this empty line?

File: core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java [hunk 2]
```diff
@@ -73,8 +77,9 @@
 
 /**
  * Unit test for {@link FileSystemMaster} when permission check is enabled by configure
- * alluxio.security.authorization.permission.enabled=true.
+ * alluxio.security.authorization.permission.enabled=true...
```
Comment (apc999) [inline core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java]: why adding two extra dots here? can you remove the extra dots?

**Comment by apc999** [comment]
@JiangXiNJU thanks for the fix. can you address the two comments i left?

**Comment by JiangXiNJU** [comment]
OK，I will soon remove them.

发自我的 iPhone

**Comment by JiangXiNJU** [comment]
Thanks for the comments，I have already addressed the two issues

**Pull request merged**